laws/src/main/scala/algebra/laws/RingLaws.scala

```diff
@@ -107,7 +107,15 @@ trait RingLaws[A] extends GroupLaws[A] {
     name = "ring",
     al = additiveCommutativeGroup,
     ml = multiplicativeMonoid,
-    parents = Seq(rig, rng)
+    parents = Seq(rig, rng),
+    "fromInt" -> forAll { (n: Int) =>
+      Ring.fromInt[A](n) ?== A.sumN(A.one, n)
+    },
+    "fromBigInt" -> forAll { (ns: List[Int]) =>
```

Comment:
okay, as long as this is not one of those "it exists but finding the homomorphism is not always trivial" type things, I'm +1.